src/plugins/intel_gna/backend/gna_limitations.cpp

```diff
@@ -168,6 +169,52 @@ void Validator::ThrowIfNotEmpty(const std::string prefix, const std::string erro
 
 } // namespace Cnn2D
 
+const std::vector<ov::element::Type> kSupportedOutputTypes = {
+    ov::element::f32,
+    ov::element::i32
+};
+
+const std::vector<ov::element::Type> kSupportedInputTypes = {
+    ov::element::u8,
```

Comment:
why `u8` instead `i8` ?

Reply:
Because officially we are supporting U8, I16 and FP32 input precisions only